src/Proto.Cluster/Identity/IIdentityStorage.cs

```diff
@@ -32,7 +32,8 @@ public SpawnLock(string lockId, ClusterIdentity clusterIdentity)
             ClusterIdentity = clusterIdentity;
         }
     }
-
+    
+    //TODO: what is this? its not used atm..
```

Comment:
Should be cleaned up, but lets do the changes it after merging the identity changes